bugzilla_dashboard/components_query.json

```diff
@@ -0,0 +1,102 @@
+{
+    "P1Defect": {
+        "label": "P1s defect",
+        "parameters": {
+            "f1": "creation_ts", "o1": "greaterthaneq", "v1": "-1y",
+            "priority": "P1",
+            "resolution": "---",
+            "bug_type": "defect"
+        }
+    },
+    "unassignedBetaBugs": {
+        "label": "Unassigned tracked beta bugs",
+        "parameters": {
+            "f1": "cf_tracking_firefox_beta",
+            "o1": "anyexact",
+            "v1": "+,blocking",
+            "f2": "cf_status_firefox_beta", "o2": "equals", "v2": "affected",
+            "f3": "assigned_to", "o3": "equals", "v3": "nobody@mozilla.org"
+        }
+    },
+    "nightlyNewBug": {
+        "label": "Nightly New Regression",
+        "hidden": "true",
+        "parameters": {
+            "keywords": "regression,",
+            "keywords_type": "allwords",
+            "resolution": "---",
+  
+            "f1": "cf_status_firefox_nightly", "o1": "equals", "v1": "affected",
+            "f2": "OP", "j2": "OR",
```

Comment:
here OP == Open Parenthesis and CP == Close Parenthesis and j2=OR means "apply OR operator to the content of parenthesis" so here f2, j2 are useless since the parenthesis contain only 1 element (f3,o3,v3).